environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EncryptionKeyArnValidator.java

```diff
@@ -0,0 +1,28 @@
+package com.sequenceiq.environment.environment.validation.validators;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.validation.ValidationResult;
+
+@Component
+public class EncryptionKeyArnValidator {
+
+    private static final Pattern ENCRYPTION_KEY_ARN_PATTERN = Pattern.compile("^arn:(aws|aws-cn|aws-us-gov):kms:[a-zA-Z0-9-]+:[0-9]+:" +
```

Comment:
I think, for AWS we should check the existence of the key directly - as APIs are available. We can check if the keyArn provided by user is present in `aws kms list-keys`.
A little background, For Azure we validated the format of key because the existence validation is not possible(at least I am not aware of) currently - CB-13810

Reply:
A separate jira is created and being worked on to validate the key existence in AWS https://jira.cloudera.com/browse/CB-13464?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel